includes/blocks/form/class-kadence-blocks-radio-block.php

```diff
@@ -83,10 +83,14 @@ public function build_html( $attributes, $unique_id, $content, $block_instance )
 
 		$radio_label = $attributes;
 		$radio_label['inputName'] = 'rb' . $class_id;
-
+		$inline_options = '';
+		if( isset($attributes['inline']) && true === $attributes['inline']) {
+			$inline_options = ' kb-radio-check-items-inline';
+		}
 		$inner_content .= '<fieldset class="kb-radio-check-item-wrap" id="' . $this->field_name( $radio_label ) . '" data-type="radio" data-required="' . $is_required . '" ' . $this->additional_fieldset_attributes( $attributes ) . '>';
 		$inner_content      .= $this->field_legend( $radio_label );
 		$inner_content      .= $this->field_aria_label( $attributes );
+		$inner_content 		.= '<div class="kb-radio-check-items-wrap' . $inline_options . '">';
```

Comment:
Is this wrap necessary? 
If it is, then I think we should only wrap if we need to for inline. Only add as much div's as we need to get the job done.

Reply:
It is only necessary if we want to use flexbox, but yes, I will made the wrap conditionally rendered. Or you think it is better to use `display: inline-block`?

Reply:
I mean, if inline looks the same / like people would expect, then I'd say go with that because it uses one less container. 
But if flex works better than stick with that.

Reply:
Got rid of the wrap in the frontend. I decided not to do the same in the backend because it implies to modify the parent div and that is related to all form fields.

Reply:
Done